src/lxc/lxc_init.c

```diff
@@ -293,3 +311,139 @@ int main(int argc, char *argv[])
 		exit(EXIT_FAILURE);
 	exit(ret);
 }
+
+
+
+static void print_usage(const struct option longopts[])
```

Comment:
Well, I would've hard-coded the usage actually and you sort of already print it in `print_help()` but there's no real reason to nack it. :)